### ISSUE TITLE
Exposure to birds

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9857]
**Preferred term label:**
Exposure to birds

**Synonyms**
Avian

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hypersensitivity pneumonitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

added as
Exposure to birds
HP:0034981
